mkdocs.yml

```diff
@@ -13,6 +13,9 @@ theme:
   palette:
     primary: 'teal'
     accent: 'teal'
+  font:
```

Comment:
In this file there are also other changes in respect to the PR merged on master. Why?